raiden-dapp/src/store/notifications/notification-context.ts

```diff
@@ -0,0 +1,6 @@
+export enum NotificationContext {
```

Comment:
These enums are really simple, maybe this and importance would better be on `types.d.ts`? If needed, that can be turned into `types.ts`

Reply:
Unfortunately they can't, there is a problem with babel and compilation fails if I put them in a file that has other types.